src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -151,20 +154,20 @@
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
     <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
     <Compile Include="$(CommonPath)\System\Data\Locale\Locale.cs">
-        <Link>System\Data\Locale\Locale.cs</Link>
+      <Link>System\Data\Locale\Locale.cs</Link>
```

Comment:
> I will take care of moving the FieldNameLookup to SqlClient in this step.

OK, and moving the Locale files as well? The reason I'm being picky about the Locale files, is if the Locale files do end up being included in multiple `System.Data.*` assemblies, then there will be multiple `CultureInfo` caches in memory for apps that use those assemblies. If the Locale files are only needed in SqlClient, it's probably better that they live there to discourage spreading to other assemblies unnecessarily.